topology/probes/k8s/graph.go

```diff
@@ -24,22 +24,58 @@ package k8s
 
 import (
 	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/topology"
 	"github.com/skydive-project/skydive/topology/graph"
 )
 
+func setManager(m *graph.Metadata) {
```

Comment:
Maps are passed as reference so there's not need to pass it as a pointer.

Besides, it's a bit overkill to create a function just to set an attribute. Maybe defining a constant for the manager is enough (KubernetesManager ?) and set it inline where setManager is called.

Reply:
agreed x 2 ... will fix accordingly